src/Common/src/System/IO/DriveInfoInternal.Unix.cs

```diff
@@ -10,6 +10,13 @@ namespace System.IO
     /// <summary>Contains internal volume helpers that are shared between many projects.</summary>
     internal static partial class DriveInfoInternal
     {
+#if MONO
+    static class Unix
+    {
+#endif
```

Comment:
I don't understand why that's an issue.  Code in Environment/EnvironmentUnix that accesses SpecialFolder can just use `Environment.SpecialFolder` rather than just `SpecialFolder`... why is that problematic?